app/src/main/res/values-it/strings.xml

```diff
@@ -123,10 +123,10 @@ Continua a praticare misure igieniche protettive e seguire le regole di comporta
     <!-- CSV wird heruntergeladen -->
     <string name="downloading_csv">"Scarica il CSV"</string>
     <string name="handshakes_heading_homescreen">" "</string>
-    <string name="handshakes_title_homescreen">"Incontri"</string>
+    <string name="handshakes_title_homescreen">"Co-presenze"</string>
     <string name="inform_heading_homescreen">" "</string>
     <string name="inform_title_homescreen">"Sospetti casi COVID-19"</string>
-    <string name="inform_text_homescreen">"Informa gli utenti che hai incontrato se presenti sintomi o se sei risultato positivo per COVID-19."</string>
+    <string name="inform_text_homescreen">"Informa gli utenti con cui sei stato co-presente se avverti sintomi o se sei risultato positivo alla COVID-19."</string>
```

Comment:
al COVID-19

Reply:
suona meglio anche per me, ma in precedenza mi è stato detto che covid-19 è la malattia per questo si dovrebbe usare il femminile. Nel caso di cambio di genere si devono controllare tutte le occorrenze

Reply:
> al COVID-19

Il sito dell'OMS in inglese non aiuta a capire. Ma in realtà, c'è una parte del web site del gov italiano che riprende le informazioni dell'OMS in italiano che recita cosi: Il Corona, la COVID-19. Penso sia doveroso rispettare la forma scelta dalla massima autorità sanitaria.